### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by higs4281**
Initial cleanup and retooling for DJango 1.8 and disclosures. 
## Additions
- Program model
- extended fields for School model
- ED api exploration script
- elasticsearch dependency and exploration script
- tests
## Removals
- haystack and solr dependencies
## Review
- @ascott1 @mistergone 
- @willbarton @kave mind checking project layout?
- @rosskarchner dropping hastack/solr make sense to you? 
## Checklist
- [x] CHANGELOG has been updated

### CODE REVIEW

File: paying-for-college/disclosures/admin.py [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
 from django.contrib import admin
-from .models import School, Alias, BAHRate, Feedback, Worksheet
+from .models import School, Program, Alias, Nickname, Contact
+from .models import BAHRate, Feedback, Worksheet
 
 
 class SchoolAdmin(admin.ModelAdmin):
-    list_display = ('primary_alias', 'school_id', 'city', 'state')
-    search_fields = ['primary_alias']
+    list_display = ('primary_alias', 'school_id', 'city', 'state', 'KBYOSS')
```
Comment (willbarton) [inline paying-for-college/disclosures/admin.py]: This is super, outrageously nit-picky, but I don't think I'd know (in fact, I don't), what `KBYOSS` means. Perhaps a comment that explains it?

File: paying-for-college/disclosures/models.py [hunk 2]
```diff
@@ -10,6 +10,12 @@ class School(models.Model):
     data_json = models.TextField()
     city = models.CharField(max_length=50)
     state = models.CharField(max_length=2)
+    accreditor = models.CharField(max_length=255, blank=True)
+    url = models.TextField(blank=True)
+    degrees_predominant = models.TextField(blank=True)
+    degrees_highest = models.TextField(blank=True)
+    operating = models.BooleanField(default=True)
+    KBYOSS = models.BooleanField(default=False)  # shopping-sheet participant
```
Comment (willbarton) [inline paying-for-college/disclosures/models.py:18]: This is super, outrageously nit-picky, but I don't think I'd know (in fact, I don't), what `KBYOSS` means. Perhaps expand a bit on the comment?

**Comment by willbarton** [comment]
I think the layout makes sense to me :+1:

File: paying-for-college/disclosures/models.py (hunk 2 above)
Comment (higs4281) [inline paying-for-college/disclosures/models.py:18]: not nit-picky. it's a carryover from the current tool: Know Before You Owe Shopping Sheet.

**Comment by higs4281** [comment]
closing this out because we're going to stick with haystack/solr for now

**Pull request merged**